### DISCUSSION TITLE
How Can I Access Documentation for Older Swiper Versions?

### DISCUSSION THREAD

**Discussion by ozknozsrt** [nolimits4web/swiper#7028] [Q&A]
Hello Swiper Community,

I'm curious about how I can access the documentation for different version numbers of Swiper while using it. Specifically, I'm interested in finding documentation for older versions, but I'm having trouble locating the direct access URL or the relevant codes.

For instance, I'd like to know if we can access documentation for older versions on the Swiper website using a URL structure similar to the following:

- https://swiperjs.com/@7.4.1
- https://swiperjs.com/v=7.4.1
- https://swiperjs.com/v/7.4.1

If it's possible to obtain this documentation from the Github repository, I'd like to learn how to access these resources and find out how to get to the documentation for older versions.

I would greatly appreciate your assistance on this matter. Thank you!

**Comment by ozknozsrt** [comment]
I've found a temporary solution to access documentation for older Swiper versions. For now, I'll be using the following link to access the relevant version:

https://web.archive.org/web/20211128115328/https://swiperjs.com/get-started

I was able to find this based on the last commit date for the respective version, which appears to be Dec 2021

**Comment by ken717w** [comment]
It's insane the docs of old versions are gone, in a project that major version number bumps so quick.

Swiper 7 released just 2 years ago and it's now already 11.

**Comment by marcelofreires** [comment]
I found the docs for some older versions of the Swiper:
- version 6 - https://v6.swiperjs.com/
- version 7 - https://v7.swiperjs.com/
- version 8 - https://v8.swiperjs.com/
- version 9 - https://v9.swiperjs.com/

**Reply by gergo888** [reply]
I need version 4 documentation for an old project.

**Reply by dnndiva** [reply]
@gergo888, found it!!!! Wayback Machine is the best. [Consider donating to them, guys!](https://archive.org/donate?origin=iawww-TopNavDonateButton).

- https://web.archive.org/web/20171209112430/http://www.idangero.us/swiper/get-started/
- https://web.archive.org/web/20171210123444/http://idangero.us/swiper/api
